cs/libdpr/samples/DprCounters/DprCounters/CounterClient.cs

```diff
@@ -12,20 +12,43 @@ namespace DprCounters
     public class CounterClient
     {
         private DprClient client;
-        private Dictionary<Worker, IPEndPoint> cluster;
-        private Thread backgroundThread;
+        private IPAddress ip;
+        Dictionary<Worker, EndPoint> cluster = new Dictionary<Worker, EndPoint>();
 
-        public CounterClient(IDprFinder dprFinder, Dictionary<Worker, IPEndPoint> cluster)
+        public CounterClient(string ip, int port)
```

Comment:
Removed and made it so that the Counter Client takes in a DPR Finder but also an IP address of the cluster since the way we made it requires this IP. IP is also no longer an optional field so we are safe from warnings you mention below.